src/assets/scripts/client/game/GameController.js

```diff
@@ -402,4 +403,14 @@ export default class GameController {
     getGameOption(optionName) {
         return this.game.option.get(optionName);
     }
+
+    /**
+     *
+     * @method getPtlLengthMultiplier
+     */
+    getPtlLengthMultiplier() {
```

Comment:
I don't like this name here; we are never applying a multiplicative factor to the length of the PTL, we're setting the length to a given value. So `getPtlLength` would be more appropriate here. It's not really as if the default is 1 and then we scale that up and down, we just change the value to whatever is specified. It's sort of a semantical thing I suppose, but still.